src/Normalizer/EntityFromUuidDenormalizer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Normalizer;
+
+use ApiPlatform\Core\Exception\ItemNotFoundException;
+use Doctrine\ORM\EntityManagerInterface;
+use Ramsey\Uuid\Uuid;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+class EntityFromUuidDenormalizer implements DenormalizerInterface
+{
+    private EntityManagerInterface $entityManager;
+
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
+
+    public function denormalize($data, $type, $format = null, array $context = [])
+    {
+        if ($object = $this->entityManager->getRepository($type)->findOneBy(['uuid' => $data])) {
+            return $object;
+        }
+
+        throw new ItemNotFoundException();
```

Comment:
Can you add an exception message. Then, we will have more details when the exception is thrown

Reply:
fixed, thanks